modules.d/01bootetc/boot-etc.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+. /lib/dracut-lib.sh
+
+bootetc=$(getarg bootetc=)
```

Comment:
OK, so we require an explicit opt-in karg like `bootetc=1`; and you see us changing FCOS to pass that by default?  Or it only gets injected when e.g. dm-multipath is in use?

Reply:
It's more like `bootetc=LABEL=....`. I envision this being an opt-in. We _could_ use it for FCOS, but I'm particularly keen on that making that decision today.